content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,122 @@
+communication and is used to establish browser-to-server and browser-to-browser connections
+between applications.
+
+WebRTC was originally designed to make audio, video, and data
+communication between browsers user-friendly and easy to implement.
+It was first developed by [Global IP Solutions (or GIPS)](http://www.gipscorp.com/).
+In 2011, GIPS was acquired by Google where the [W3C](https://www.w3.org/) started to work
+on a standard for WebRTC.
+
+It serves as a good choice for applications that need built-in support
+for media communication and do not have specific requirements for the underlying
+transport protocol.
+
+## WebRTC in libp2p
+
+In libp2p, WebRTC is used to connect from browsers to other nodes. libp2p WebRTC only uses
+WebRTC data channels and does not make use of any of the multimedia features.
+
+### Key features
+
+- Peer connections: WebRTC enables
+  [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes.
+
+- Data channels: WebRTC provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send a text or binary data over an active connection to a peer.
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/)) to connect to nodes that run behind
+  NATs and firewalls. In non-decentralized WebRTC, this can be facilitated by a
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
+  Using a custom signaling protocol or a different signaling service is also possible.
+  Overall, this allows for faster and more efficient communication.
+
+- Security: WebRTC connections are encrypted using
+  [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security).
+
+Browsers expose an API to establish WebRTC connections. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate.
+
+### Browser-to-Server
+
+libp2p WebRTC enables browsers to connect to public server nodes without those
+endpoints providing a TLS certificate within the browser's trustchain.
+
+{{< alert icon="" context="info" text="When connecting to a WebSocket server, browsers require the server to present a TLS certificate signed by a trusted CA (certificate authority). Few nodes have such a certificate, which is the reason that WebSocket never saw widespread adoption in the libp2p network. WebRTC and WebTransport support encrypted communication without requiring a signed certificate from a trusted CA." />}}
```

Comment:
```suggestion
{{< alert icon="" context="info" text="When connecting to a WebSocket server, browsers require the server to present a TLS certificate signed by a trusted certificate authority (CA). Few libp2p nodes meet this requirement, primarily because it's hard to get a certificate in a decentralized manner. This is the reason that WebSocket never saw widespread adoption in the libp2p network. WebRTC (and [WebTransport](#comparing-webrtc-and-webtransport)) supports encrypted communication without requiring a signed certificate from a trusted CA." />}}
```